src/main/java/com/wolox/training/models/User.java

```diff
@@ -30,15 +30,17 @@
 
   @Column(nullable = false, unique = true)
   @NotNull
-  private String userName;
-
+  private String username;
+  @Column(nullable = false)
+  @NotNull
+  private String password;
```

Comment:
Password should not have read access, i mean when you get all users this prop should not appear